app/src/main/res/drawable/ic_find_search_20_a05.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="20dp"
+    android:height="20dp"
+    android:viewportWidth="20"
+    android:viewportHeight="20">
+  <path
+      android:pathData="M9,2C5.134,2 2,5.134 2,9C2,12.866 5.134,16 9,16C10.645,16 12.157,15.433 13.352,14.483L16.634,17.766C16.947,18.078 17.453,18.078 17.766,17.766C18.078,17.453 18.078,16.947 17.766,16.634L14.483,13.352C15.433,12.157 16,10.645 16,9C16,5.134 12.866,2 9,2ZM3.6,9C3.6,6.018 6.018,3.6 9,3.6C11.982,3.6 14.4,6.018 14.4,9C14.4,11.982 11.982,14.4 9,14.4C6.018,14.4 3.6,11.982 3.6,9Z"
+      android:fillColor="?attr/daxColorPrimaryIcon"
+      android:fillAlpha="0.5"
```

Comment:
Shouldn't we apply the alpha directly in the `ImageView` widget? I feel it's not efficient to have different vector images for each configuration (size, alpha, color, etc.)